solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -1048,12 +1062,50 @@ contract WalletRegistryGovernance is Ownable {
             walletRegistry.dkgParameters().seedTimeout,
             walletRegistry.dkgParameters().resultChallengePeriodLength,
             walletRegistry.dkgParameters().resultSubmissionTimeout,
-            newSubmitterPrecedencePeriodLength
+            newSubmitterPrecedencePeriodLength,
+            walletRegistry.dkgParameters().resultChallengeExtraGas
         );
         dkgSubmitterPrecedencePeriodLengthChangeInitiated = 0;
         newSubmitterPrecedencePeriodLength = 0;
     }
 
+    /// @notice Begins the DKG result challenge extra gas update process.    .
```

Comment:
Unnecessary spaces and a dot at the end of the line :)

Reply:
f337b2f7ae864a91261bb8fdcc957d22a2b12f58